ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Update.scala

```diff
@@ -206,7 +206,7 @@ object Update {
       reason: RejectionReason,
   ) extends Update {
     override def description: String = {
-      s"Reject command ${submitterInfo.commandId}: $reason"
+      "transaction rejection"
```

Comment:
I think it wouldn't hurt to log the details. Otherwise the description makes an assumption on where it can be meaningfully used (i.e. only in an environment that uses logging context).

Reply:
We don't use it anywhere atm.
The details really are just a repetition of the logging context in a less organized manner. I think if we have made that investment to log using context, let's just stick to it.

Reply:
These descriptions used to show up in one trace log in the entire code-base and they looked awful

Reply:
fwiw we have an extension of the logging context which adds a type parameter exactly to solve this issue: if you don’t have the right context it simply won’t compile https://github.com/digital-asset/daml/blob/adeba206594b8ec3437efa99a44c4a5572d3ec6e/libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LoggingContextOf.scala#L10.

Reply:
Thanks for the pointer. The story doesn't end here for us. Tudor has started on another PR that makes the logging context usage in indexer more robust. That's where we will switch to LoggingContextOf